report/verification/mms/radioactive_decay.md

```diff
@@ -0,0 +1,209 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.2
+kernelspec:
+  display_name: vv-festim-report-env
+  language: python
+  name: python3
+---
+
++++ {"vscode": {"languageId": "javascript"}}
```

Comment:
Is this required?

Reply:
no, it's a remnant from a mistake I made, if you spot it elsewhere feel free to remove it